.travis/install_ctverif.sh

```diff
@@ -29,8 +29,9 @@ INSTALL_DIR=$1
 cd $INSTALL_DIR
 
 #install smack
-git clone --depth 1 https://github.com/smackers/smack.git -b develop
+git clone https://github.com/smackers/smack.git -b develop
```

Comment:
That was the advice I got from the SMACK developers.  I'll talk with them about doing a proper release we can consume, but until then fixing to a particular git commit should make the travis builds more reproducible.